src/main/java/eu/decentsoftware/holograms/api/actions/ClickType.java

```diff
@@ -3,7 +3,7 @@
 import org.bukkit.event.block.Action;
 
 public enum ClickType {
-    LEFT, RIGHT, SHIFT_LEFT, SHIFT_RIGHT;
+    LEFT, RIGHT, SHIFT_LEFT, SHIFT_RIGHT,HOVER,HOVER_LEAVE;
```

Comment:
Not a fan of a non-click action (just looking at a page/line) being in a enum specifically for click types.

Better separate it.

Reply:
Thank you for your reply,But I think hovering the mouse over the hologram should also be considered a ClickType. If we put them into actions, it would lose its meaning, as it would be triggered by LEFT_CLICK or RIGHT_CLICK instead of when the mouse hovers over the hologram.

While I acknowledge that my programming skills may have room for improvement, I am open to further discussion and finding a solution that best aligns with the project's needs. What are your thoughts on this approach?

Reply:
I'm mixed feelings on this to be honest.
On one hand is it nice that you want to add hover support, but on the other hand do I feel like this may not be as good in terms of performance...

Reply:
Sorry, it does indeed have performance issues; perhaps it's an immature submission. I hope this PR can remain open for others to see, maybe there are better approaches from others.